src/server.test.ts

```diff
@@ -1,35 +1,53 @@
+import request from 'supertest';
+import { app } from './server';
+import testData from './components/ContributionsEpic.testData';
+import { configuredTests } from './api/contributionsApi.testData';
+
 jest.mock('./api/contributionsApi', () => {
     return {
-        fetchDefaultEpicContent: jest.fn().mockImplementation(() =>
-            Promise.resolve({
-                heading: 'Heading',
-                paragraphs: ['Paragraph'],
-                highlighted: ['Highlight'],
-            }),
-        ),
+        fetchConfiguredEpicTests: jest
+            .fn()
+            .mockImplementation(() => Promise.resolve(configuredTests)),
     };
 });
 
-import request from 'supertest';
-import { app } from './server';
-import testData from './components/ContributionsEpic.testData';
+let oldEnableDynamicTestsConfig: string | undefined;
+
+beforeEach(() => {
+    oldEnableDynamicTestsConfig = process.env.ENABLE_DYNAMIC_TESTS;
+    process.env.ENABLE_DYNAMIC_TESTS = 'true';
+});
+
+afterEach(() => {
+    process.env.ENABLE_DYNAMIC_TESTS = oldEnableDynamicTestsConfig;
+});
 
 describe('POST /epic', () => {
     it('should return an epic when targeting is positive', async () => {
-        const { tracking, localisation, targeting } = testData;
+        const { pageTracking, targeting } = testData;
 
         const res = await request(app)
             .post('/epic')
             .send({
-                tracking,
-                localisation,
+                tracking: pageTracking,
                 targeting,
             });
 
         expect(res.status).toEqual(200);
         expect(res.body).toHaveProperty('data');
         expect(res.body.data).toHaveProperty('html');
         expect(res.body.data).toHaveProperty('css');
+        expect(res.body.data).toHaveProperty('meta');
```

Comment:
Would it not be easier to test the whole `meta` object in one assertion?

Reply:
Yeah that's a good idea. thanks. I like the failure message you get when comparing strings because it's so obvious. But given this object is small I think it would be easy to follow either way. I'll make the change :+1: